src/radius/common.h

```diff
@@ -0,0 +1,159 @@
+/**
+ * @file
+ * @author Alexandru Mereacre
+ * @date 2022
+ * @copyright
+ * SPDX-FileCopyrightText: © 2022 NQMCyber Ltd and edgesec contributors
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ * @brief File containing the common definitions used by radius and eap
```

Comment:
I feel like at least some of the code in this file is taken from another source.

Can you add copyright/licensing information from that project too?